### PR TITLE
[cli] transfer:erc20 and balance commands

### PR DESCRIPTION
**Pull request by gastonponti**
### Description

Command to transfer and check balance of erc20 contracts

### Tested

Manually in alfajores

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/cli/src/commands/account/balance.ts [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
+import { Args, Flags } from '../../utils/command'
 
 export default class Balance extends BaseCommand {
-  static description = 'View Celo Dollar and Gold balances for an address'
+  static description = 'View Celo Stables and CELO balances for an address'
 
   static flags = {
     ...BaseCommand.flags,
+    erc20Address: Flags.address({
+      description: "Custom erc20 to check it's balance too",
```
Comment (yorhodes) [inline packages/cli/src/commands/account/balance.ts]:
```suggestion
      description: "Address of generic ERC-20 token to also check balance for",
```

File: packages/cli/src/commands/transfer/erc20.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx, failWith } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class TransferErc20 extends BaseCommand {
+  static description = 'Transfer ERC20 to a specified address'
+
+  static flags = {
+    ...BaseCommand.flags,
+    erc20Address: Flags.address({
```
Comment (yorhodes) [inline packages/cli/src/commands/transfer/erc20.ts:15]: we can define this flag once and reuse it across the commands

File: packages/sdk/contractkit/src/base.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ export enum CeloContract {
   DowntimeSlasher = 'DowntimeSlasher',
   Election = 'Election',
   EpochRewards = 'EpochRewards',
+  ERC20 = 'ERC20',
```
Comment (yorhodes) [inline packages/sdk/contractkit/src/base.ts:9]: this seems like the wrong place for this to live

Comment (yorhodes) [reply packages/sdk/contractkit/src/base.ts:9]: is "ERC20" a `CeloContract`?

**Review by yorhodes — APPROVED** [review]

File: packages/sdk/contractkit/src/base.ts (hunk 3 above)
Comment (gastonponti) [reply packages/sdk/contractkit/src/base.ts:9]: We should define what we think it is a CeloContract. In a way, we think of those as the contracts supported/maintained by celo. The ERC20 is a standard accepted by everyone.
Let's say that the contract does not exist in the ethereum network, we would probably want to handle and support that contract.
So, in a way, it is something standard for celo too, we are just taking the advantage of reuse it from other network.

If we think of the CeloContracts as only the ones that live on the registry, we should also remove the multisig from that list, for example.

If we think of the CeloContracts, as the one accepted by the community, all the ercXXX accepted, should be supported too

File: packages/cli/src/commands/transfer/erc20.ts (hunk 2 above)
Comment (gastonponti) [reply packages/cli/src/commands/transfer/erc20.ts:15]: Don't know if we require this in more that these two commands. We could leave it this way and if we see another pattern, we could extract it in the future

**Pull request merged**